d3/tql.js

```diff
@@ -0,0 +1,1002 @@
+// -----------------------------------------------------------------------------
+//
+// This file is the copyrighted property of Tableau Software and is protected
```

Comment:
The overall project license is MIT. This conflicting copyright should be removed.

Reply:
Removed.